packages/gas-report/ts/index.ts

```diff
@@ -205,13 +205,16 @@ function saveGasReport(gasReport: GasReport, path: string) {
 
 function readStdIn(): Promise<string> {
   return new Promise((resolve) => {
+    console.log("Waiting for stdin...");
     let data = "";
 
     process.stdin.on("data", (chunk) => {
+      console.log(chunk.toString());
       data += chunk;
     });
 
     process.stdin.on("end", () => {
+      console.log("Done waiting for stdin");
```

Comment:
`process.stdin` is a stream so you could also

```
process.stdin.pipe(process.stdout)
```

Reply:
and could prob lean on a generic stream package like https://github.com/stream-utils/stream-to-array

```
const stdin = (await toArray(process.stdin)).map((chunk) => Buffer.from(chunk).toString('utf8'))
```